db/migrate/20210217111153_add_patient_id_index_to_bp_mat_views.rb

```diff
@@ -0,0 +1,18 @@
+class AddPatientIdIndexToBpMatViews < ActiveRecord::Migration[5.2]
+  disable_ddl_transaction!
+
+  def change
+    add_index :latest_blood_pressures_per_patient_per_months,
+      :patient_id,
+      name: "index_latest_bp_per_patient_per_months_patient_id",
+      algorithm: :concurrently
+    add_index :latest_blood_pressures_per_patient_per_quarters,
+      :patient_id,
+      name: "index_latest_bp_per_patient_per_quarters_patient_id",
+      algorithm: :concurrently
+    add_index :latest_blood_pressures_per_patients,
+      :patient_id,
+      name: "index_latest_bp_per_patient_patient_id",
+      algorithm: :concurrently
```

Comment:
Do we know that these indexes are being used? At some point (not now surely), the cost of having these indexes will add up.

Reply:
Use https://github.com/nilenso/postgresql-monitoring#index_usage